From 3f7cba58132c2d27714a5c9a76768a244758f534 Mon Sep 17 00:00:00 2001
From: Luis Medinas <lmedinas@gnome.org>
Date: Fri, 03 Dec 2010 13:46:41 +0000
Subject: Fix Bug 635614 - tomboy insecure LD_LIBRARY_PATH

Fix CVE-2010-4005. Originally found by Ludwig Nussel <lnussel@novell.com>.
---
diff --git a/Tomboy/tomboy-panel.in b/Tomboy/tomboy-panel.in
index 206a21f..0caeee0 100644
--- a/Tomboy/tomboy-panel.in
+++ b/Tomboy/tomboy-panel.in
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-export LD_LIBRARY_PATH="@pkglibdir@${LD_LIBRARY_PATH+:$LD_LIBRARY_PATH}"
+export LD_LIBRARY_PATH="@pkglibdir@${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}"
 export MONO_PATH=$MONO_PATH:@pkglibdir@:@pkglibdir@/addins
 export TOMBOY_WRAPPER_PATH="@bindir@/@wrapper@"
 TOMBOY_CONFIG_DIR="${XDG_CONFIG_HOME:-$HOME/.config}/tomboy"
diff --git a/Tomboy/tomboy.in b/Tomboy/tomboy.in
index 4512746..a69ddb1 100644
--- a/Tomboy/tomboy.in
+++ b/Tomboy/tomboy.in
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-export LD_LIBRARY_PATH="@pkglibdir@${LD_LIBRARY_PATH+:$LD_LIBRARY_PATH}"
+export LD_LIBRARY_PATH="@pkglibdir@${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}"
 export MONO_PATH=$MONO_PATH:@pkglibdir@:@pkglibdir@/addins
 export TOMBOY_WRAPPER_PATH="@bindir@/@wrapper@"
 
--
cgit v0.8.3.4
